core/src/main/java/org/elasticsearch/search/aggregations/InternalAggregation.java

```diff
@@ -70,7 +70,8 @@ public Type(String name, BytesReference stream) {
         }
 
         /**
-         * @return The name of the type (mainly used for registering the parser for the aggregator (see {@link org.elasticsearch.search.aggregations.Aggregator.Parser#type()}).
+         * @return The name of the type of aggregation.  This is the key for parsing the aggregation from XContent and is the name of the
```

Comment:
Rewrote the javadoc because it failed the build. This is why it is a very good idea to use `{@link}` and `{@linkplain}` liberally. They fail the build if you make them invalid.